mlbench/refimpls/pytorch/optim/lr.py

```diff
@@ -250,3 +258,8 @@ def get_scheduler(options, optimizer):
         return multistep_learning_rates_with_warmup(options, optimizer)
     else:
         raise NotImplementedError
+
+
+if __name__ == '__main__':
+    import doctest
```

Comment:
There's already unittesting with tox, so that might be a better place for tests. Then it also gets tested when on the build server.